docs/admin/install.md

```diff
@@ -4,20 +4,20 @@ Here you will find detailed instructions on how to **install** and **configure**
 
 ## Prerequisites
 
-- The plugin relies on [`nautobot_plugin_nornir`](https://pypi.org/project/nautobot-plugin-nornir/) to be installed and both plugins to be enabled in your configuration settings.
-- The latest version of this plugin is compatible with Nautobot 2.0.0 and higher, see [this dedicated page](compatibility_matrix.md) for a full compatibility matrix and the deprecation policy.
+- The app relies on [`nautobot_plugin_nornir`](https://pypi.org/project/nautobot-plugin-nornir/) to be installed and both apps to be enabled in your configuration settings.
```

Comment:
To validate `nautobot-plugin-nornir` is the actual package name and even though we're doing a rename of that project as well, we're not planning on renaming that package name during this time.

Reply:
Not at this time. Since it would be a breaking change I would like to plan out the best plan of action before renaming the python package.

Reply:
Leave everything pointing to nautobot_plugin_nornir and Nautobot Plugin Nornir for now, it's going to be a pain to get that fixed, and would prefer to get this merged quicker.